docassemble/EFSPIntegration/efm_client.py

```diff
@@ -367,9 +368,10 @@ def update_service_contact(self, service_contact_id, **kwargs):
         data=json.dumps(kwargs))
     return self._call_proxy(send)
 
-  def create_service_contact(self, service_contact):
+  def create_service_contact(self, service_contact:Individual):
+    service_contact_dict = serialize_person(service_contact)
     send = lambda: self.proxy_client.post(self.base_url + f'firmattorneyservice/service-contacts', 
```

Comment:
I'm actually having some troubles with how this is designed. The actual dictionary needs to have some service contact specific information in it:
* isPublic
* isInFirmMaster
* administrativeCopy (separate from email address)

But the serialize person step strips those out.